### PR TITLE
added clustercleanade

### PR DESCRIPTION
**Pull request by Creatorbot01**
## About the PR
added a brand new shiny clustercleanade

## Why / Balance
to make cleaning up major and i mean major mess easier

## Media
image

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: ClusterCleanade now available in janitorial crate
- remove: mopbucket from janitorial crate
- tweak: janibelt can now bring clustercleanade and util belt now can carry engi goggles in them

### CODE REVIEW

**Comment by Creatorbot01** [comment]
what why rsi validator just die??? wtf

**Comment by TheDoctor1977** [comment]
mop bucket removed from jani crate thank god
now you can actually close the damn thing after you open it

**Comment by Erisfiregamer1** [comment]
Please atempt to fix the grammar in the description before I explode.

File: Resources/Prototypes/Entities/Objects/Weapons/Throwable/clusterbang.yml [hunk 1]
```diff
@@ -323,3 +323,38 @@
...
     containers:
       cluster-payload: !type:Container
+
+
+
+- type: entity
+  parent: GrenadeBase
+  id: ClusterBangCleanade
+  name: cluster cleanade
+  description: Launches three cleanade. when it too messy for one cleanade how about 3 cleanade.(janitor.co does not responsible any mass slipping accident with this products)
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Entities/Objects/Weapons/Throwable/clusterbang.yml:333]:
```suggestion
  description: Launches three cleanades after the timer runs out.
```
this is parity with the clusterbang description

File: Resources/Textures/Objects/Weapons/Grenades/clusterclean.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation and modified by Swept at https://github.com/tgstation/tgstation/commit/29c0ed1b000619cb5398ef921000a8d4502ba0b6 and modified by Swept",
```
Comment (TheDoctor1977) [inline Resources/Textures/Objects/Weapons/Grenades/clusterclean.rsi/meta.json:4]: the listed commit does not contain this sprite

**Review by TheDoctor1977 — CHANGES_REQUESTED** [review]

**Comment by bruhmogus** [comment]
cl should be
add: Clustercleanade now available in janitorial crates.
remove: Mop buckets will no longer spawn in janitorial crates, allowing them to be closed.
tweak: Janitor belts can carry clustercleanades, and utility belts can now hold engineering goggles.

**Comment by Erisfiregamer1** [comment]
PR is based on prebase code, redo your PR on the new code please.

**Pull request closed without merging**